src/Packagist/WebBundle/Resources/config/algolia_settings.yml

```diff
@@ -1,22 +1,22 @@
 searchableAttributes:
-    - "name"
-    - "package_name"
-    - "package_organisation"
-    - "description"
-    - "tags"
+  - "unordered(package_name)"
+  - "unordered(tags)"
+  - "unordered(package_organisation)"
+  - "description"
 
 ranking:
     - "typo"
     - "geo"
     - "words"
     - "filters"
+    - "desc(popularity)"
```

Comment:
Not sure about that one, making it above `attributes` will put very popular libraries with a match in the `description` to be above a "perfect" match in the `package_name` for instance.

Reply:
We wanted to do this to highlight package that uses a techno but aren't named after it. For example if you look for "websocket", we want Ratchet lib to come before any random package with _websocket_ in the name. Another example would be if you look for "templating", you'll find twig, smarty and mustache, which are probably what you're looking for.

Also popularity is only from 0 to 10 so it should create a lot of ties.

Maybe I can put a webpage up with these settings so everyone can try easily 💡

Reply:
> Also popularity is only from 0 to 10 so it should create a lot of ties.

That's a good thing, but I still think we want to "boost" popular objects only if they have a 9 or 10.

What about:
 - keeping `desc(popularity)` in the `customRanking`
 - creating a new `most_popular` boolean attribute in each record  with value `1` if popularity is 10 or 9, `0` otherwise
 - replacing this `desc(popularity)` middle-ranking criterion by `desc(most_popular)`
 - eventually moving this criterion at the very top of the ranking formula (before `typo`)